src/components/KDropdownMenu/KDropdownMenu.vue

```diff
@@ -170,6 +174,16 @@ const handleSelection = (item: DropdownItem): void => {
   selectedItem.value = item
 }
 
+const handleCloseDropdown = async (): Promise<void> => {
+  hidePopover.value = true
+
+  // reset the hidePopover value so it's ready for the next time the dropdown is opened
+  // need nextTick to ensure the popover is hidden before resetting the value
+  await nextTick(() => {
```

Comment:
Even though you’re now awaiting this promise, when called from the event emit it will still not actually await before moving on. 

This may not matter if it appears to work, but pointing out that this won’t actually be called asynchronously.

Reply:
It appears to work, should be fine